### PR TITLE
Support matplotlib

### PR DESCRIPTION
**Pull request by Asaaj**
# Blocked by:
- [ ] #998

### CODE REVIEW

File: quadratic-kernels/python-wasm/quadratic_py/matplotlib_patch.py [hunk 1]
```diff
@@ -0,0 +1,48 @@
...
+    import matplotlib.pyplot
+    import matplotlib.backend_bases
+    import matplotlib.backends.backend_agg
+
+    @matplotlib.backend_bases._Backend.export
+    class _MatplotlibIntercepter(matplotlib.backend_bases._Backend):
+        class _InterceptingManager(matplotlib.backend_bases.FigureManagerBase):
+            def __init__(self, *args, **kwargs):
+                super().__init__(*args, **kwargs)
+                print("CREATED")
```
Comment (Asaaj) [inline quadratic-kernels/python-wasm/quadratic_py/matplotlib_patch.py:33]: There is still an issue that multiple runs of a `matplotlib` cell cause an assertion failure, as it thinks we're trying to return multiple figures from one cell. I haven't tracked this down yet

**Comment by davidkircos** [comment]
matplotlib via pyodide doesn't support png yet, so this can't be used yet

**Comment by davidkircos** [comment]
Apparently it does work now, reopening

**Comment by davidkircos** [comment]
Outdated, closing for now. Linked in related issue https://github.com/quadratichq/quadratic/issues/1654 to revive when prioritized.

**Pull request closed without merging**